keyboards/k_type/k_type.h

```diff
@@ -14,29 +14,29 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-#ifndef K_TYPE_H
-#define K_TYPE_H
+
+#pragma once
 
 #include "quantum.h"
 
-#define KEYMAP( \
-    K01, K02, K03, K04, K05, K06, K07, K08, K09, K10, K11, K12, K13, K14, K15, K16, \
-    K17, K18, K19, K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K30, K31, K32, K33, \
-    K34, K35, K36, K37, K38, K39, K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, K50, \
-    K51, K52, K53, K54, K55, K56, K57, K58, K59, K60, K61, K62, K63, \
-    K64, K65, K66, K67, K68, K69, K70, K71, K72, K73, K74, K75, K76, \
-    K77, K78, K79, K80, K81, K82, K83, K84, K85, K86, K87 \
+#define XXX KC_NO
+
+#define LAYOUT_tkl_ansi( \
+    k00,      k10, k20, k30, k40, k50, k60, k70, k80, k90, k01, k11, k21,      k31, k41, k51, \
+    k61, k71, k81, k91, k02, k12, k22, k32, k42, k52, k62, k72, k82, k03,      k23, k33, k43, \
+    k53, k63, k73, k83, k93, k04, k14, k24, k34, k44, k54, k64, k74, k84,      k05, k15, k25, \
+    k35, k45, k55, k65, k75, k85, k95, k06, k16, k26, k36, k46,      k66, \
+    k86,      k07, k17, k27, k37, k47, k57, k67, k77, k87, k97,      k18,           k38, \
+    k48, k58, k68,                k78,                k88, k98, k09, k19,      k29, k39, k49 \
 ) { \
-    {   K01,   K11,   K21,   K30,   K39,   K48,   K58,   K65, KC_NO,   K83, }, \
-    {   K02,   K12,   K22, KC_NO,   K40,   K49,   K59,   K66,   K75,   K84, }, \
-    {   K03,   K13,   K23,   K31,   K41,   K50,   K60,   K67, KC_NO,   K85, }, \
-    {   K04,   K14,   K24,   K32,   K42,   K51,   K61,   K68,   K76,   K86, }, \
-    {   K05,   K15,   K25,   K33,   K43,   K52,   K62,   K69,   K77,   K87, }, \
-    {   K06,   K16,   K26,   K34,   K44,   K53, KC_NO,   K70,   K78, KC_NO, }, \
-    {   K07,   K17,   K27,   K35,   K45,   K54,   K63,   K71,   K79, KC_NO, }, \
-    {   K08,   K18,   K28,   K36,   K46,   K55, KC_NO,   K72,   K80, KC_NO, }, \
-    {   K09,   K19,   K29,   K37,   K47,   K56,   K64,   K73,   K81, KC_NO, }, \
-    {   K10,   K20, KC_NO,   K38, KC_NO,   K57, KC_NO,   K74,   K82, KC_NO, }, \
+    { k00, k01, k02, k03, k04, k05, k06, k07, XXX, k09 }, \
```

Comment:
So, this is completely changed... and I believe this is what's making my board not work.

I've included a photo of the back of the board, which has all the switch numbers, and how I named everything originally. Maybe there's a difference between revisions? The official kll files list what I used for the scancodes and should be able to be used to verify this. https://github.com/kiibohd/controller/blob/master/Scan/K-Type/scancode_map.kll

![IMG_0490](https://user-images.githubusercontent.com/107097/88841824-5aae6e00-d193-11ea-97ee-1df6803d5b51.jpg)

Reply:
There should be no functional difference here. All I've done is lowercase the `K`s and renumber them to reflect the electrical row/columns.

Reply:
Ah, you're right. When I did my spot check, it looked like a few were messed up. Any reason for the renumbering? I think when I did it based on the numbers on the back of the board (which I believe are related to the scancodes used in the official firmware?) to make it easier to debug.

Reply:
Current QMK convention is to use the electrical row/column number, but they can really be named anything - it's just to map between the top and bottom halves of the layout macro.